test/controllers/notification/NotificationCallbackControllerSpec.scala

```diff
@@ -32,19 +34,38 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.xml.NodeSeq
 
 
-class NotificationCallbackControllerSpec extends PlaySpec with MockitoSugar with ScalaFutures {
-  
+class NotificationCallbackControllerSpec extends PlaySpec with MockitoSugar with ScalaFutures with SpecBase {
+
   val mockNotificationService = mock[NotificationService]
-  val controller = new NotificationCallbackController(mockNotificationService)
+  val controller = new NotificationCallbackController(mockNotificationService, appConfig)
```

Comment:
as this is a unit test we should really be using a mock AppConfig with the auth token retreival mocked.